torch/csrc/jit/codegen/cuda/scheduler.h

```diff
@@ -13,20 +14,68 @@ TORCH_CUDA_API bool scheduleFusion(
     Fusion* fusion,
     const at::ArrayRef<c10::IValue> inputs);
 
+/**
```

Comment:
Are we using doxygen style comments? (not a bad idea to start doing it, but please stick to // comments even for multi-line)